pkg/leap/leap-file_test.go

```diff
@@ -299,3 +299,65 @@ func Test_handleLeapIndication(t *testing.T) {
 	assert.Equal(t, "3928780800", lm.leapFile.LeapEvents[1].LeapTime)
 	assert.Equal(t, "4291747200", lm.leapFile.ExpirationTime)
 }
+
+func Test_New_Good(t *testing.T) {
```

Comment:
Do we need to do any cleanup after this test case?

Reply:
Good one, done